File: SWEA/SWEA_7732.cpp
// algorithm study
// SWEA_[D3]7732_시간 개념

#include <iostream>

using namespace std;

int main() {
    int test_case;
    cin >> test_case;
    for (int t = 0; t < test_case; t++) {
        int h, m, s, now_time = 0, next_time = 0, time[3];
        char c;
        cin >> h >> c >> m >> c >> s;
        now_time += h * 3600 + m * 60 + s;
        cin >> h >> c >> m >> c >> s;
        next_time += h * 3600 + m * 60 + s;
        if (next_time < now_time)
            next_time += 24 * 3600;
        time[0] = (next_time - now_time) / 3600;
        time[1] = ((next_time - now_time) % 3600) / 60;
        time[2] = ((next_time - now_time) % 3600) % 60;
        cout << "#" << t + 1 << " ";
        for (int i = 0; i < 3; i++) {
            if (time[i] < 10)
                cout << 0 << time[i];
            else
                cout << time[i];
            if (i != 2)
                cout << ":";
            else
                cout << "\n";
        }
    }
}

File: SWEA/D3/SWEA_7732.py
"""
algorithm study
SWEA_[D3]7732_시간 개념
"""
test_case = int(input())
for i in range(test_case):
    now_time = input()
    next_time = input()

    now_time = int(now_time[:2]) * 3600 + int(now_time[3:5]) * 60 + int(now_time[6:])
    next_time = int(next_time[:2]) * 3600 + int(next_time[3:5]) * 60 + int(next_time[6:])
    if now_time > next_time:
        next_time += 24 * 3600
    answer = "%02d:%02d:%02d" % (
        (next_time - now_time) / 3600,
        ((next_time - now_time) % 3600) / 60,
        ((next_time - now_time) % 3600) % 60,
    )
    print("#" + str(i + 1) + " " + answer + "\n")
